### ISSUE TITLE
Kepler.gl website does not load

### ISSUE THREAD

**Issue by ctriouCVNA** [keplergl/kepler.gl#1943]
When navigating to Kepler.gl I'm presented with a blank page and the following error 

**To Reproduce**
1. Go to https://kepler.gl
2. See blank page and error in console
```
bundle.js:437 Uncaught TypeError: Object(...) is not a function
    at Module.<anonymous> (bundle.js:437:164295)
    at n (bundle.js:1:110)
    at Module.<anonymous> (bundle.js:437:120156)
    at n (bundle.js:1:110)
    at Module.<anonymous> (bundle.js:437:119199)
    at n (bundle.js:1:110)
    at Module.<anonymous> (bundle.js:498:471575)
    at n (bundle.js:1:110)
    at Object.<anonymous> (bundle.js:498:471470)
    at n (bundle.js:1:110)
(anonymous) @ bundle.js:437
n @ bundle.js:1
(anonymous) @ bundle.js:437
n @ bundle.js:1
(anonymous) @ bundle.js:437
n @ bundle.js:1
(anonymous) @ bundle.js:498
n @ bundle.js:1
(anonymous) @ bundle.js:498
n @ bundle.js:1
(anonymous) @ bundle.js:1
(anonymous) @ bundle.js:1
kepler.gl/:1 Unchecked runtime.lastError: The message port closed before a response was received.
```

**Expected behavior**
Webpage loads without a blank screen

**Additional context**

**Comment by Niko-La** [comment]
Saved maps from Dropbox url are not loading too. 

🙏

**Comment by bj-liang** [comment]
I ran the demo app local on my computer
The current version [v2.5.5] failed and gives an error code per the image below. 
![Screenshot 2022-08-19 214509](https://user-images.githubusercontent.com/64744472/185724578-0cee4e82-3f9b-4474-8f7f-35e653530734.png)

Then I ran a previous version, it worked fined. [v2.5.2]

**Comment by abcnishant007** [comment]
This is still not working, any fixes?!

**Comment by davidcalhoun** [comment]
Looks like the fix is in #1944

**Comment by ctriouCVNA** [comment]
Thank you!
